site/profile/files/presentation/styles.css

```diff
@@ -44,4 +44,14 @@ input[type=button] {
 iframe {
   border: none;
   overflow: hidden;
-}
\ No newline at end of file
+}
+
+.error {
```

Comment:
At some point, we should establish a standard color scheme for all our tooling and sites.